File: typescript/strategy/abc-bank-remit.ts
import { Fee } from "./fee/fee";
import { Remit } from "./remit";

export class AbcBankRemit extends Remit {
    constructor(fee: Fee) {
        super(fee);
    }

    public createTransferMessage(): string {
        return "Hey Abc Bank! I will give you money! OK?";
    }
}

File: src/main/java/strategy/AbcBankRemit.java
package strategy;

import strategy.fee.Fee;

public class AbcBankRemit extends Remit {
    public AbcBankRemit(Fee fee) {
        super(fee);
    }

    @Override
    public String createTransferMessage() {
        return "Hey Abc Bank! I will give you money! OK?";
    }

}
